os/board/common/README.md

```diff
@@ -0,0 +1,78 @@
+# How to partition a flash
+
+This document provides a way to partition in flash. Use CONFIG when flash partition.  
```

Comment:
Please rephrase as "This document describes the method to partition flash memory using TizenRT's menuconfig."

Reply:
Thanks. I modified it.